documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsVisibilityCell.vue

```diff
@@ -92,11 +93,19 @@ export default {
         }
       }
       return false;
+    },
+    sharedDocumentSuspendedLabel(){
+      return this.$t('documents.label.share.document.suspend');
     }
   },
+  created() {
+    this.$transferRulesService.getDocumentsTransferRules().then(rules => {
+      this.sharedDocumentSuspended = rules.sharedDocumentStatus === 'true';
+    });
+  },
   methods: {
     changeVisibility() {
-      if (!this.file.acl.canEdit) {
+      if (!this.file.acl.canEdit || this.sharedDocumentSuspended) {
```

Comment:
Could you also check if the suspend is considered from backend ?

Reply:
done !